unitary/examples/quantum_rpg/main_loop.py

```diff
@@ -66,6 +67,14 @@ def party(self):
     def file(self):
         return self.game_state.file
 
+    def print_status(self):
+        for idx, qar in enumerate(self.game_state.party):
+            print(
+                f"{idx+1}) {qar.name} {qar.class_name},\tqaracter sheet:",
```

Comment:
Should all of this except the `idx` be delegated to a `status` method on the `qaracter`? Making this 
```print(f"{idx+1}) {qar.status}", file=...) for idx, qar in enumerate(self.game_state.party)```